pkg/lib/operation/template/local/repository/describe/operation.go

```diff
@@ -0,0 +1,64 @@
+package describe
+
+import (
+	"github.com/keboola/keboola-as-code/internal/pkg/log"
+	"github.com/keboola/keboola-as-code/internal/pkg/template"
+)
+
+type dependencies interface {
+	Logger() log.Logger
+}
+
+func Run(tmpl *template.Template, d dependencies) (err error) {
+	logger := d.Logger().InfoWriter()
```

Comment:
to je troska zavadzajuce, `logger` -> `w` / `writer`?

Reply:
No jo. Nevšiml jsem si, fixnu v tom druhým pr 🙂

Reply:
https://github.com/keboola/keboola-as-code/pull/674/commits/caed738dc23ee30c4ce1d9a4f3abf20b4aa9e58d